content/v2.0/reference/cli/influx/pkg/stack/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: influx pkg stack
+description: The 'influx pkg stack' command and its subcommands manage InfluxDB stacks.
+menu:
+  v2_0_ref:
+    name: influx pkg stack
+    parent: influx pkg
+weight: 101
+v2.0/tags: [templates]
+---
+
+The `influx pkg stack` command manages InfluxDB stacks.
```

Comment:
maybe define stacks within context of templates?

Reply:
I'm going to save this for the task based version of this doc. Once that exists, I'll add it as a related link to this doc. This update is laying down the foundation for stacks and they're still likely to change a lot. My goal here is to just document the CLI. We can cover the conceptual stuff later.